XamlControlsGallery/ControlPages/AcrylicPage.xaml.cs

```diff
@@ -44,5 +49,11 @@ private void FallbackColorSelector_SelectionChanged(object sender, SelectionChan
             Rectangle shape = (sender == FallbackColorSelectorInApp) ? CustomAcrylicShapeInApp : CustomAcrylicShape;
             ((AcrylicBrush)shape.Fill).FallbackColor = ((SolidColorBrush)e.AddedItems.First()).Color;
         }
+
+        private void LuminositySlider_ValueChanged(object sender, RangeBaseValueChangedEventArgs e)
+        {
+            Rectangle shape = CustomAcrylicShapeLumin;
+            ((AcrylicBrush)shape.Fill).TintLuminosityOpacity = e.NewValue;
```

Comment:
Do we need to wrap this in an API check so that it doesn't crash down-level? Or does the property still exist down-level, but it degrades gracefully (ie. is ignored or something)?

Reply:
Just checked! If running on RS5 or lower, Acrylic will use the old recipe even if the new property is explicitly defined.

Reply:
Thanks, sounds good!